.github/workflows/ci.yml

```diff
@@ -49,17 +49,18 @@ jobs:
       fail-fast: true
       matrix:
         os:
-          - { label: ubuntu-20.04, code: focal }
+          - { label: ubuntu-latest, code: latest }
```

Comment:
Is there any reason why this is using ubuntu and not debian unstable?

Reply:
I believe the reason was that debian unstable (and indeed any debian release) is simply not available in GitHub-hosted runners.

If you're asking bearing in mind the presence of newer versions of LLVM in debian unstable, we can simply install them from https://apt.llvm.org/ if we decide to use them in CI.

Reply:
> I believe the reason was that debian unstable (and indeed any debian release) is simply not available in GitHub-hosted runners.

That shouldn't be relevant in the first place, see how i do it in rawspeed.

Reply:
You asked me about the reason why this particular OS image was chosen, I gave my understanding (this is just my impression, it was not me who wrote the CI scripts). Actually, the choice of OS doesn't really matter, so I don't see any point in discussing it.

Reply:
> You asked me

FWIW, no. That generally should be read as "hmm, this looks odd. consider whether this should be changed."

Reply:
Going in the right direction, ubuntu-latest is better as a testing version than a specific release.